manifests/webapp/prefer_related_applications.json

```diff
@@ -4,7 +4,7 @@
       "prefer_related_applications": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/Progressive_web_apps/Manifest/Reference/prefer_related_applications",
-          "spec_url": "https://w3c.github.io/manifest/#prefer_related_applications-member",
+          "spec_url": "https://wicg.github.io/manifest-incubations/index.html#prefer_related_applications-member",
```

Comment:
Moved into Manifest Incubations.

I'm not sure if this means it's deprecated or non-standard? (I guess not.)